### DISCUSSION TITLE
Presets showcase on the website

### DISCUSSION THREAD

**Discussion by 0xMRTT** [GradienceTeam/Gradience#379] [Announcements]
We added a new page on the website with a list of presets with a screenshot for each one. 

👉 https://gradienceteam.github.io/presets 👈

We also improved the preset submission process. If you found any bugs, things that can be improved, submit an issue.

**Comment by jdamboeck** [comment]
Page not found.

**Reply by daudix** [reply]
Yeah, the website is under construction, should be done soon
